vercel.json

```diff
@@ -1,5 +1,16 @@
 {
 	"github": {
 		"silent": true
-	}
+	},
+	"headers": [
+		{
+			"source": "/images/(.*)",
+			"headers": [
+				{
+					"key": "Cache-Control",
+					"value": "public, max-age=3600"
+				}
+			]
+		}
```

Comment:
looks like this doesn't fix the issue. this might also be a bad idea in terms of vercel network transfer costs (because we are fetching the image without optimization)

we could try the next.js optimized version (eg. https://www.highlight.io/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fsession-replay.abf97e7c.webp&w=3840&q=75&dpl=dpl_J79fBF8iLhW8eEuULM8LwRdrHcBM) but that still feels hacky. ideally, could we bundle these images into the frontend build so they are served from `app.highlight.io/...`?

https://www.loom.com/share/2f8ddaaa01834915aa92660d0a4d28fa

Reply:
ah, saw the note `Note: new icons won't be cached until new landing page is deployed` - makes sense since the header isn't applied yet.

Reply:
still, i would be curious to see if it's easy to host the images directly from app.highlight.io

Reply:
Yeah, I can try and move them to app.highlight.io, since I don't believe they are being used on the highlight.io site either

Reply:
@Vadman97 I moved the images to app.highlight.io - its working locally, but TBD if anything needs to be done for render/reflame